dataplane/loader/src/main.rs

```diff
@@ -62,6 +102,7 @@ async fn main() -> Result<(), anyhow::Error> {
         .context("failed to attach the egress TC program")?;
 
     info!("starting api server");
+    info!("Using tls config: {:?}", &opt.tls_config);
```

Comment:
:+1:

Reply:
I don't love the way rust destructures (or doesn't, rather) the config when printing it, but that's a small nit I can live with for now.